documents/ta-protocols/ta-protocol-2021-10-08.md

```diff
@@ -0,0 +1,42 @@
+# Meeting Agenda
+
+Group: OOPP-WITH-THE-BOIS
+
+Date: 2021-10-08
+
+Chair: Pelle
+
+Participants: Martin, Jonathan, Jacob, Mathias, Pelle
+
+## Objectives
+
+* No objectives.
+
+## Reports
+
+* No Reports.
+
+## Discussion items
+
+1. What grammatical tense should SDD and RAD be written in?
+1. Should refactor tasks be in user stories?
+1. Should our peer-review be in the final report or their review on our project?
+1. More than one instance of model in application? i.e Colony and World(Environment). Or should they be wrapped as one
+   instance?
+1. Should we remove all fields and methods from class diagrams (only keeping them in interfaces)?
+1. Can we have more than 3 paragraphs in each section? (Show current RAD)
+1. How do we reference buttons in RAD? “New Game” button? “New Game”-button? new game button?
+
+## Outcomes and assignments
+
+1. Present tense works, as long as you are consistent.
+1. Nope
+1. Both peer reviews should be included to be contained.
+1. Application can instantiate multiple parts of the model, that is okay.
+1. Everything that is not private should be visible.
+1. It’s allowed.
+1. Doesn’t really matter, as long as we are consequent.
```

Comment:
Seems I made a bad word choice when trying to quickly write down Pelle's answers, this should be "consistent" and not "consequent".

Reply:
Was not you, I added the answer since you left it empty and I did some bad translation from swedish to english "konsekvent" is consistent and not concequent. Making the changes.